src/hotspot/share/runtime/frame.hpp

```diff
@@ -433,8 +433,8 @@ class frame {
   void describe_pd(FrameValues& values, int frame_no);
 
  public:
-  void print_value() const { print_value_on(tty,nullptr); }
-  void print_value_on(outputStream* st, JavaThread *thread) const;
+  void print_value() const { print_value_on(tty); }
+  void print_value_on(outputStream *st) const;
```

Comment:
Nit: the * should be at the end of `outputStream` not the start of `st`.

Reply:
Thanks! I missed that IDE "helped" me here.